common/mq/publisher.js

```diff
@@ -12,8 +13,14 @@ const log = require('@idearium/log')();
 const publish = (type, data) => {
 
     return manager.publish(type, data)
-        .then(() => log.debug({ data, type }, `Published message of type: ${type}`))
-        .catch(err => log.error({ err }, `Could not publish message of type: ${type}`));
+        .then(() => multiLog(
+            {
+                trace: { data },
+                info: { type },
+            },
+            'Published message'
+        ))
+        .catch((err) => log.warn({ err, type }, 'Could not publish message'));
```

Comment:
This is serious enough to remain at error isn't it?

```suggestion
        .catch((err) => log.error({ err, type }, 'Could not publish message'));
```